spacetemplate/assets/scrum.yaml

```diff
@@ -0,0 +1,242 @@
+space_template:
+  id: "629c963a-174c-4c37-b487-272067e88bd4"
+  name: Scrum
+  description: >
+    Scrum is an agile framework for managing work with 
+    an emphasis on software development.
+work_item_types:
+- id: &commonWorkItemTypeID "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Common Work Item Type
+  description: TBD
+  fields:
+    "system.history": 
```

Comment:
@baijum @DhritiShikhar please don't start names with `system.`. This was only done because the old fields from planner item have a special meaning to the system.

Reply:
@baijum @DhritiShikhar also please don't just add fields as you go but think about each individually. What do you think the history is for a work item? To me it is what @baijum implements as the timeline and it has its own API even. So no need to add history as a field here.

Reply:
@kwk I have already told you that these commits are not the final implementation. So, please give me some time before you start reviewing. The data in the excel sheet is very confusing and it's very difficult to make sense of all the data. My first step was to map the data in the excel sheet to yaml. And then I can add/remove/modify as per the requirements. Since you were reviewing this PR actively. I made sure to inform you beforehand.